l10n/en/firefox/switch.ftl

```diff
@@ -5,7 +5,9 @@
 ### URL: https://www-dev.allizom.org/firefox/switch/
 
 switch-switch-from-chrome = Switch from { -brand-name-chrome } to { -brand-name-firefox } in just a few minutes
+switch-switching-to-firefox-is-fast-updated = Switching to { -brand-name-firefox } is fast, easy and risk-free, because { -brand-name-firefox } imports your bookmarks, passwords and preferences from { -brand-name-chrome }.
```

Comment:
@peiying2 what's the recommended process for adding a new string and deprecating an older one?

Here I'm removing the word "autofill" from an existing string. Should I add a comment to convey that the old string is obsolete?

Also note I'm updating a string below to use the brand name for Facebook like you mentioned earlier.

Reply:
If we don't have a tagging process in place, ideally we have a new ID with the changed string. We just remove it and add a new ID/string. I am not sure it is good practice to use the same ID to update the string. Let me check what has been done with other fluent projects.

Reply:
@alexgibson New ID for revised string.